bin/psalm

```diff
@@ -14,7 +14,8 @@ $options = getopt(
     [
         'help', 'debug', 'config:', 'monochrome', 'show-info:', 'diff',
         'file:', 'self-check', 'update-docblocks', 'output-format:',
-        'find-dead-code', 'init', 'find-references-to:', 'root:', 'threads:'
+        'find-dead-code', 'init', 'find-references-to:', 'root:', 'threads:',
+        'report-xml:', 'report-json:', 'report-emacs:'
```

Comment:
Could we shorten this just to `--report=filename.xml` and use the `--output-format=xml` to specify the type. That way the list of commands is more compact and composable.

Reply:
That was how I write it first, but the `--output-format` will be use for two different things (and it force the console output to be the same as the report output).  
For example, the XML output is nice for external tool (like a **ci**) but not really readable while the `console` output is the opposite.

I guess a can add a `--report-format` option (and we lose the ability to have multiple report, but I don't really see usage of it)

Reply:
Ohhh. So you have a scenario where you want to output multiple types of report at the same time?

Reply:
I can't find any real usage of a multiple report output.  
I guess it can be useful if you send your result into different _post processing/aggregator_ tool that don't read the same type of data (a java based tool is more likely to use XML, where a Javascript one prefer to use JSON)

This options is available on PHPMD, PHPUnit, phploc, phpcs, PhpMetrics. (It's mainly because it's found on several tools that I came up with this solution)

Reply:
`phpcs` doesn't seem to have separate commands for different file formats (though PHPUnit does)